### PR TITLE
Add Brazilian Translations

### PR DESCRIPTION
**Pull request by ejkun**
I am targeting this branch, because this PR does not cause a BC

## Changelog
```markdown
### Fixxed
- Added brazilian translations for color names and date_range messages.
```
## Subject
Added the following missing brazilian translations
* date_range_start
* date_range_end
* All colors

### CODE REVIEW

**Comment by core23** [comment]
Please run `make lint` on the new translation file @ejkun

**Comment by ejkun** [comment]
Okay, just a sec.

File: Resources/translations/SonataCoreBundle.pt_BR.xliff [hunk 1]
```diff
@@ -28,575 +28,575 @@
...
             </trans-unit>
             <trans-unit id="forestgreen">
                 <source>forestgreen</source>
-                <target>forestgreen</target>
+                <target>Verde floresta</target>
             </trans-unit>
             <trans-unit id="fuchsia">
                 <source>fuchsia</source>
-                <target>fuchsia</target>
+                <target></target>
```
Comment (jordisala1991) [inline Resources/translations/SonataCoreBundle.pt_BR.xliff]: Why does this color does not have translation?

Comment (ejkun) [reply Resources/translations/SonataCoreBundle.pt_BR.xliff]: My mistake, too many files open, missed one translation.

**Comment by core23** [comment]
Thanks @ejkun

**Comment by ejkun** [comment]
Just a note, some colors don't have a translation. Like Pink, Pink is "Rosa", but "Rosa" here has a different HexCode.

Another thing, if possible, could this PR become a release? I have and know many projects that use SonataColorPicker. This PR is extremely important.

**Comment by Soullivaneuh** [comment]
Fix**x**ed? Please watch out before merge. :warning:

**Pull request merged**